modules/ztp-nbde-generating-a-new-tang-server-key.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-nbde-implementation-guide.adoc
+
+[id="ztp-nbde-generating-a-new-tang-server-key_{context}"]
+= Generating a new Tang server key
+
+.Prerequisites
+
+* A root shell on the Linux machine running the Tang server.
+
+* To facilitate verification of the Tang server key rotation, encrypt a small test file with the old key:
++
+[source,terminal]
+----
+# echo plaintext | \ clevis encrypt tang '{"url":"http://localhost:7500”}' -y \ >/tmp/encrypted.oldkey
```

Comment:
Should there be `\` in here?

Reply:
This might be a copy&paste issue.  The Google doc where this originated had line breaks and used `\` at the end for continuations.  This might be improperly formatted now.

Reply:
Removed the / characters. Now looks like this: 
# echo plaintext | clevis encrypt tang '{"url":"http://localhost:7500”}' -y >/tmp/encrypted.oldkey